### PR TITLE
[hi] Add content/hi/docs/setup/learning-environment/_index.md

### PR DESCRIPTION
**Pull request by ghost**
Localised the website/content/en/docs/setup/learning-environment/_index.md to Hindi langugage.

### CODE REVIEW

**Review by sftim — DISMISSED** [review]
@Darshnadas I suspect that  you might have misunderstood the advice to localizers in the original page.

There is a Netlify redirect from https://kubernetes.io/docs/setup/learning-environment/ to https://kubernetes.io/docs/tasks/tools/

This English page content only exists to provide a navigation stub and to protect in case that redirect is one day removed. 

As you're localizing that page, you only need to copy the front matter (the first 4 lines) *and* add a redirect into `/static/_redirects`:
- redirect `/hi/docs/setup/learning-environment/` to https://kubernetes.io/docs/tasks/tools/

[Later,] if you also localize https://kubernetes.io/docs/tasks/tools/, you can change that redirect so that it targets https://kubernetes.io/hi/docs/tasks/tools/

**Comment by Darshnadas** [comment]
@sftim Ohh, okay.

File: content/hi/docs/setup/learning-environment/_index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+शीर्षक:  सीखने का माहौल
+वजन: 20
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:3]:
```suggestion
title:  सीखने का माहौल
weight: 20
```

File: content/hi/docs/setup/learning-environment/_index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+इसलिए `kind`, [`minikube`](https://minikube.sigs.k8s.io/) iएक उपकरण है जो आपको कुबेरनेट्स चलाने देता है
+स्थानीय स्तर पर `।` minikube`c`आपके व्यक्तिगत पर एकल-नोड कुबेरनेट्स क्लस्टर चलाता है
+कंप्यूटर (विंडोज़, मैक और लिनक्स पीसी सहित) ताकि आपताकि आप कोशिश कर सकें
+कुबेरनेट्स, या दैनिक विकास कार्य के लिए।
+
+
+आप अधिकारी का अनुसरण कर सकते हैं 
+[शुरू हो जाओ!] (https://minikube.sigs.k8s.io/docs/start/) gमार्गदर्शन करें यदि आपका ध्यान है
+उपकरण स्थापित होने पर।पर।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:34]:
```suggestion
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/learning-environment/_index.md (hunk 2 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/learning-environment/_index.md:34]: Can you add a redirect to `/static/_redirect` as mentioned by @sftim in previous comment

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/setup/learning-environment/_index.md

**Comment by anubha-v-ardhan** [comment]
@Darshnadas Can you implement the requested changes. We could move towards merging this PR.

**Comment by anubha-v-ardhan** [comment]
@Darshnadas Are you still working on this PR? Can you have a look at the suggestions please.

**Comment by Darshnadas** [comment]
@anubha-v-ardhan Hi, I just checked the comments and will work on the PR.

**Comment by Darshnadas** [comment]
@anubha-v-ardhan Updated the PR according to the changes, please check it.

File: static/_redirects [hunk 3]
```diff
@@ -506,6 +506,7 @@
 /docs/setup/learning-environment/     /docs/tasks/tools/  302!
 /id/docs/setup/learning-environment/     /id/docs/tasks/tools/  302!
 /zh/docs/setup/learning-environment/     /zh/docs/tasks/tools/  302!
+/hi/docs/setup/learning-environment/ /hi/docs/tasks/tools/ 302!
```
Comment (anubha-v-ardhan) [inline static/_redirects:509]:
```suggestion
/hi/docs/setup/learning-environment/     /hi/docs/tasks/tools/ 302!
```

File: static/_redirects [hunk 4]
```diff
@@ -542,4 +543,7 @@
 /docs/concepts/configuration/pod-overhead/     /docs/concepts/scheduling-eviction/pod-overhead/   301
 /id/docs/concepts/configuration/pod-overhead/     /id/docs/concepts/scheduling-eviction/pod-overhead/   301
 
+
 /pt/*         /pt-br/:splat  302!
+
+
```
Comment (anubha-v-ardhan) [inline static/_redirects:549]:
```suggestion
/pt/*         /pt-br/:splat  302!
```

File: content/hi/docs/setup/learning-environment/_index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+{{/* If you're localizing this page, you only need to copy the front matter */}}
+{{/* and add a redirect into "/static/_redirects", for YOUR localization. */}}
+-->
+
+## kind
+
+[`kind`](https://kind.sigs.k8s.io/docs/) आपको Kकुबेरनेट्स को अपने स्थानीय कंप्यूटर पर चलाने देता है। इस
+- उपकरण के लिए आवश्यक है कि आपके पास 
+-[Docker](https://docs.docker.com/get-docker/)स्थापित और कॉन्फ़िगर किया गया।
+-
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:20]:
```suggestion
[`kind`](https://kind.sigs.k8s.io/docs/) आपको कुबेरनेट्स को अपने स्थानीय कंप्यूटर पर चलाने देता है। इस 
उपकरण के लिए आवश्यक है कि आपके पास 
[Docker](https://docs.docker.com/get-docker/) स्थापित और कॉन्फ़िगर किया गया हो।

```

File: content/hi/docs/setup/learning-environment/_index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+-kind [Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start)पृष्ठ
+-आपको दिखाता है कि आपको काम करने के लिए क्या करना हैक्या करना हे
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:22]:
```suggestion
kind [Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start) पृष्ठ
आपको दिखाता है कि काम करने के लिए क्या क्या करना है। 
```

File: content/hi/docs/setup/learning-environment/_index.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+## minikube
+-
+-इसलिए `kind`, [`minikube`](https://minikube.sigs.k8s.io/) iएक उपकरण है जो आपको कुबेरनेट्स चलाने देता है
+-स्थानीय स्तर पर `।` minikube`c`आपके व्यक्तिगत पर एकल-नोड कुबेरनेट्स क्लस्टर चलाता है
+-कंप्यूटर (विंडोज़, मैक और लिनक्स पीसी सहित) ताकि आपताकि आप कोशिश कर सकें
+-कुबेरनेट्स, या दैनिक विकास कार्य के लिए।
+-
+-
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:31]:
```suggestion

`kind` की तरह, [`minikube`](https://minikube.sigs.k8s.io/) एक टूल है जो आपको स्थानीय स्तर पर 
कुबेरनेट्स चलाने देता है। `minikube` आपके कंप्यूटर (Windows, MacOS और Linux) पर एकल-नोड 
कुबेरनेट्स क्लस्टर चलाता है ताकि आप कुबेरनेट्स का परीक्षश 
या दैनिक विकास कार्य के लिए प्रयोग कर सकें।


```

File: content/hi/docs/setup/learning-environment/_index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+-आप अधिकारी का अनुसरण कर सकते हैं 
+-[शुरू हो जाओ!] (https://minikube.sigs.k8s.io/docs/start/) gमार्गदर्शन करें यदि आपका ध्यान है
+-उपकरण स्थापित होने पर।पर।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/learning-environment/_index.md:34]:
```suggestion
टूल इंस्टॉल करने के लिए आप 
[शुरू करें!](https://minikube.sigs.k8s.io/docs/start/) गाइड 
का अनुसरण कर सकते हैं। 
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
@Darshnadas Can you implement all the suggestions, Seems like only some of them were done :)

**Comment by Darshnadas** [comment]
@anubha-v-ardhan Okay, I'll update it

**Comment by Darshnadas** [comment]
@anubha-v-ardhan okay I'll update the changes.

**Comment by Darshnadas** [comment]
@anubha-v-ardhan Updated.

File: content/hi/docs/setup/learning-environment/_index.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+{{/* If you're localizing this page, you only need to copy the front matter */}}
+{{/* and add a redirect into "/static/_redirects", for YOUR localization. */}}
+-->
+
+## kind
+
+[`kind`](https://kind.sigs.k8s.io/docs/) आपको Kकुबेरनेट्स को अपने स्थानीय कंप्यूटर पर चलाने देता है। इहै।इस
+ उपकरण के लिए आवश्यक है कि आपके पास 
+[Docker](https://docs.docker.com/get-docker/)स्थापित और कॉन्फ़िगर किया गकिया गया हो।
+
```
Comment (Garima-Negi) [inline content/hi/docs/setup/learning-environment/_index.md:20]: 1. Kकुबेरनेट्स typo can be fixed.
2. another fix can be made for `इहै।इस` 
3. `गकिया` seems like an extra word here in the sentence

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: static/_redirects (hunk 4 above)
Comment (sftim) [inline static/_redirects:549]: If we can: omit all changes here

**Review by sftim — COMMENTED** [review]
For `content/hi/docs/setup/learning-environment/_index.md` **no translation of the running text is required** so long as the correct redirect is configured.

> If you're localizing this page, you only need to copy the front matter
> and add a redirect into "/static/_redirects", for YOUR localization.

File: static/_redirects (hunk 4 above)
Comment (Darshnadas) [reply static/_redirects:549]: omit as in should remove everything and just keep the title table?

**Comment by sftim** [comment]
/label tide/merge-method-squash

If you know how to squash commits here @Darshnadas, please do. It's handy but not strictly required.

**Comment by Darshnadas** [comment]
> /label tide/merge-method-squash
> 
> If you know how to squash commits here @Darshnadas, please do. It's handy but not strictly required.

Okay @sftim :)

**Comment by sftim** [comment]
BTW, I'm still seeing multiple commits. A squash would mean there was 1 commit.

File: static/_redirects (hunk 4 above)
Comment (sftim) [reply static/_redirects:549]: The only change to _this_ file should be to add one line:
```diff
  /zh/docs/setup/learning-environment/     /zh/docs/tasks/tools/  302!
+ /hi/docs/setup/learning-environment/     /hi/docs/tasks/tools/ 302!
  /docs/setup/learning-environment/kind/  /docs/tasks/tools/  302
```

**Comment by anubha-v-ardhan** [comment]
@Darshnadas It would be nice if you can implement https://github.com/kubernetes/website/pull/29416#discussion_r771453096 before squashing :)

**Comment by Darshnadas** [comment]
okay

**Comment by Darshnadas** [comment]
okay @anubha-v-ardhan

File: static/_redirects (hunk 4 above)
Comment (anubha-v-ardhan) [inline static/_redirects:549]:
```suggestion
/pt/*         /pt-br/:splat  302!
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
@Darshnadas You can commit the suggestion

File: static/_redirects (hunk 4 above)
Comment (Darshnadas) [reply static/_redirects:549]: okay

**Comment by anubha-v-ardhan** [comment]
/remove-tide/merge-method-squash

**Comment by anubha-v-ardhan** [comment]
/assign
/lgtm
/approve

Thank you @Darshnadas :)

**Comment by sftim** [comment]
Looks like this was squashed OK (I've been away for yule).

**Comment by anubha-v-ardhan** [comment]
@sftim This PR seems all set now. If the changes look good to you, can we merge this.

**Comment by sftim** [comment]
No objections from me.

**Comment by anubha-v-ardhan** [comment]
@sftim I meant can you /approve. 
I can't due to changes on static/_redirects

**Comment by Darshnadas** [comment]
Thank you so much @sftim

**Pull request merged**